### ISSUE TITLE
[Moderate] The decorative image is not hidden from screen readers

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#621]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87011 |
| **Checkpoint** | 1.1 Text Alternatives › Alternative Text (Decorative Images) (1.1.1.d) |
| **Impact** | Moderate |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 04/21/26 at 11:16 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The decorative image is not hidden from screen readers.
On the Files screen, in the Drag and Drop area, the graphic is labeled and doesn't need to be. See Create Job User Experience video at 16:20.

## Recommendation

Rule : 
Images that do not convey content, are decorative, or are redundant to content that is already conveyed in text MUST be coded to be ignored by screen readers.

How To Fix: 
Fix this issue by using ONE of the following techniques:
1. Remove the title attribute and use only an empty alt attribute (alt="") on the <img> element.

<a href="contact.html"><img src="phone.png" alt="">Contact Us</a>

2. Use one of the following ARIA attributes on the on the <img> element: role="none" or aria-hidden="true". NOTE: This technique will allow you to use a non-empty alt attribute value if you want to supply alt text for SEO purposes.

<a href="contact.html"><img src="phone.png" role="none">Contact Us</a>

<a href="contact.html"><img src="phone.png" aria-hidden="true">Contact Us</a>

3. Turn the image into a CSS background image.

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

Background: 
Decorative images do not add information to the content of a page. Examples of decorative images include an image whose content is provided by adjacent text, or an image that is used just for visual effect such as a hero image, a background image, a stock photo, or a decorative flourish. Decorative images should be coded in such a way that screen readers can ignore them.

## Screenshots

![03_decorative](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87011/03_decorative.png)
